src/main/resources/assets/farmersdelight/lang/ko_kr.json

```diff
@@ -113,28 +122,35 @@
   "item.farmersdelight.fried_rice": "볶음밥",
   "item.farmersdelight.pumpkin_soup": "호박 수프",
   "item.farmersdelight.baked_cod_stew": "대구 스튜",
+  "item.farmersdelight.noodle_soup": "우동",
 
+  "item.farmersdelight.honey_glazed_ham": "꿀 바른 돼지고기 구이",
   "item.farmersdelight.pasta_with_meatballs": "미트볼 파스타",
   "item.farmersdelight.pasta_with_mutton_chop": "양고기 파스타",
+  "item.farmersdelight.roasted_mutton_chops": "구운 양고기 요리"
```

Comment:
You forgot a comma on this line (see red-highlight colon below).

Reply:
Just added comma, Thx!